### DISCUSSION TITLE
What URL do I patch it with?

### DISCUSSION THREAD

**Discussion by configari** [LittleBigRefresh/Refresh#15] [Q&A/Support] [answered]
I was looking to play-test this and I was confused whilist patching, it said that I should not patch it with 10061 port then what URL do I patch this with?

**Comment by Beyley** [comment, marked as answer]
an example URL to patch to would be `http://localhost:10061/lbp` but replace `localhost` with the name of your server, if you are on real hardware that is
if you are on RPCS3 and your server is on the same machine as your game, then you should be able to use `localhost`

**Reply by configari** [reply]
I patched it and I can't seem to connect to the servers.

**Reply by Beyley** [reply]
could you elaborate on your setup? are you on RPCS3 or real hardware

**Reply by configari** [reply]
RPCS3, I'm using the releases exe. I can compile if its needed.

**Reply by configari** [reply]
If I load that URL on my browser it just says not found.

**Reply by jvyden** [reply]
That means the server is working - it's receiving your request and responding with a 404.

**Reply by configari** [reply]
Well I tried patching it and I couldn't go online, I got the EBOOT file from another folder that had LBP assets. I think thats why I can't go online, but the patching is successful.

**Reply by jvyden** [reply]
The EBOOT should be under `dev_hdd0/game/<titleId>/USRDIR`. You may also have to manually specify the EBOOT to launch through the File -> Boot SELF menu.

**Reply by configari** [reply]
I have did those and I still couldn't connect, perhaps my game is broken?

**Comment by jvyden** [comment]
> it said that I should not patch it with 10061 port

This warning message is unclear - it was written before a massive change in how Bunkum (our underlying server architecture) works and no longer applies. I'll rewrite this warning shortly.

**Reply by jvyden** [reply]
I've addressed this in [210417b](https://github.com/LittleBigRefresh/Docs/commit/210417ba83f336128b0a9b72c3ed1313c999f898) and [3f23099](https://github.com/LittleBigRefresh/Bunkum/commit/3f230991be3a73d6c224bc279cb674f706971462).
